IntercomDotnet/Resources/Users.cs

```diff
@@ -47,7 +47,7 @@ public dynamic DeleteWithUserId(string userId)
         [Obsolete("userId is a string in the Intercom API so please use the string-based overload going forward.")]
         public dynamic Get(string email = null, int? userId = null)
         {
-            return Get(email, userId?.ToString());
+            return Get(email, userId.ToString());
```

Comment:
userId?.ToString() will throw a compilation error, fixed this with Convert.ToString()